deployment/script/build-CCNP.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -o errexit
+
+ccnp_url=https://github.com/intel/confidential-cloud-native-primitives.git
+device_plugin=docker.io/library/ccnp-device-plugin:0.1
+get_quote=docker.io/library/ccnp-quote-server:0.1
+measurement=docker.io/library/ccnp_measurement_server:0.1
+eventlog=ccnp_eventlog_server:0.1
```

Comment:
1. Why some of the image tags have "docker.io" prefix, some not?
2. docker.io image might have "rate limit" issue when pulling, shall we consider "quay.io" instead for an open source project choice?